trident.md

```diff
@@ -18,9 +18,9 @@ To instead install the latest development version you can follow these steps:
 ## Guide for trident
 
 ### Poseidon package repositories
-Trident generally requires Poseidon datasets to work with. Most trident subcommands therefore have a central parameter, called `--baseDir` or simply `-d` to specify a base directory with Poseidon packages. For example, if all Poseidon packages live inside a repository at `/path/to/poseidon/packages` you would simply say `trident <subcommand> -d /path/to/poseidon/dirs/` and `trident` would automatically search all subdirectories inside of the repository for valid poseidon packages.
+Trident generally requires Poseidon datasets to work with. Most trident subcommands therefore have a central parameter, called `--baseDir` or simply `-d` to specify one or more base directories to look for Poseidon packages. For example, if all Poseidon packages live inside a repository at `/path/to/poseidon/packages` you would simply say `trident <subcommand> -d /path/to/poseidon/dirs/` and `trident` would automatically search all subdirectories inside of the repository for valid poseidon packages (as identified by valid `POSEIDON.yml` files).
 
-We typically recommend arranging a poseidon repository in a hierarchical way. For example:
+You can recommend arranging a poseidon repository in a hierarchical way. For example:
```

Comment:
"You can recommend" is probably not what you meant?

Reply:
No indeed. Thanks.